qmk_cli/helpers.py

```diff
@@ -30,7 +30,10 @@ def broken_module_imports():
     """Make sure we can import all the python modules.
     """
     broken_modules = find_broken_requirements('requirements.txt')
-    broken_dev_modules = find_broken_requirements('requirements-dev.txt') if cli.config.user.developer else []
+    broken_dev_modules = []
+    if cli.config.user.developer:
+        if len(sys.argv) == 1 or (len(sys.argv) > 1 and 'config' != sys.argv[1]):
```

Comment:
If someone has specified global flags (such as --no-color) this will break.

```suggestion
        args = sys.argv[1:]

        while args[0][0] != '-':
            del args[0]

        if args[0] == 'config':
```

Reply:
Thx! I did not consider that.